talaiot/src/main/kotlin/com/cdsap/talaiot/publisher/InfluxDbPublisher.kt

```diff
@@ -164,8 +163,10 @@ class InfluxDbPublisher(
                 report.environment.gradleVersion?.let { addField("gradleVersion", it) }
                 report.environment.gitBranch?.let { addField("gitBranch", it) }
                 report.environment.gitUser?.let { addField("gitUser", it) }
+                report.environment.hostname?.let { addField("hostname", it) }
+                report.environment.osManufacturer?.let { addField("osManufacturer", it) }
```

Comment:
Question: do we need to include these into the other publishers as well? e.g. [`ElasticSearchPublisher`](https://github.com/cdsap/Talaiot/blob/environment_metrics_fix/talaiot/src/main/kotlin/com/cdsap/talaiot/publisher/ElasticSearchPublisher.kt#L97)

Reply:
Question: since these are part of the default configuration, should they be included in the [`ExecutionReport`](https://github.com/cdsap/Talaiot/blob/environment_metrics_fix/talaiot/src/main/kotlin/com/cdsap/talaiot/entities/ExecutionReport.kt#L78)?

Reply:
First, thank you, really appreciate your help.  
Done, update ElasticSearchPublisher. 
I opened an issue #158, to unify as provider the SimpleMetrics. This will remove the duplicated code and will be easier to test. Let me know if you are interested.

Then, to your second question, I think these fields are related to the environment execution of the build, that's why are in the Default as environment.

And finally, I applied your commit for the test. 
Thanks!

Reply:
Glad to help! Thank you for the updates.

> Then, to your second question, I think these fields are related to the environment execution of the build, that's why are in the Default as environment.

Sorry, I think I was not clear. My question is regarding whether we should initialize those environment properties in the `ExecutionReport` as well, in the same way, we are doing in here. 

The [`flattenBuildEnv()`](https://github.com/cdsap/Talaiot/blob/environment_metrics_fix/talaiot/src/main/kotlin/com/cdsap/talaiot/entities/ExecutionReport.kt#L51) function seems to do that for some of the values:

```kotlin
...
environment.osVersion?.let { map["osVersion"] = it }
environment.javaVmName?.let { map["javaVmName"] = it }
environment.cpuCount?.let { map["cpuCount"] = it }
environment.username?.let { map["username"] = it }
environment.gradleVersion?.let { map["gradleVersion"] = it }
...
```

Maybe https://github.com/cdsap/Talaiot/issues/158 will solve this "issue" as well. :thinking:

Reply:
Sorry, I got confused. `flattenBuildEnv()` caches the values, not initializes them.